Allow user to show / hide battery percentage

zats opened the pull request:
App reads menubar's preferences to decide whether percentage should be shown
this also removed the logic of hiding battery percentage if charing and battery is not full

zats committed:
Make percentage optional

uliwitness commented:
I don't think this should be an explicit option. I'll see if I can add code that checks whether the OS is set to showing the percentage, and use that info instead.

zats commented:
You are absolutely right. Removed configuration, replaced with reading battery menubar's preferences instead. Not sure how to observe it for changes though (if value changes while helper is running). Observing for key didn't seem to work for me
